scheduler/kubernetes/support.md

```diff
@@ -55,8 +56,11 @@ If the PVC creation is failing, this could be due the following reasons
 ### Useful commands
 * List PX pods: `kubectl get pods -l name=portworx -n kube-system`
 * Describe PX pods: `kubectl describe pods -l name=portworx -n kube-system`
-* Logs of all PX pods: `kubectl logs -l name=portworx -n kube-system --tail=99999`
-* Follow logs of a particular px pod: `kubectl logs -w <pod-name> -n kube-system`
+* Get PX cluster status: `kubectl exec $(kubectl get pods -l name=portworx -n kube-system -o jsonpath='{.items[0].metadata.name}') -n kube-system -- /opt/pwx/bin/pxctl status`
+* List PX volumes: `kubectl exec $(kubectl get pods -l name=portworx -n kube-system -o jsonpath='{.items[0].metadata.name}') -n kube-system -- /opt/pwx/bin/pxctl volume list`
```

Comment:
These look very scary.  I'd suggest to break them up into multiple commands, ie:

```
px_pod=$(kubectl get pods -l name=portworx -n kube-system -o jsonpath='{.items[0].metadata.name}')
kubectl exec $px_pod -n kube-system -- /opt/pwx/bin/pxctl status
kubectl exec $px_pod -n kube-system -- /opt/pwx/bin/pxctl volume list
```